src/sentryLogger/sentryLog.ts

```diff
@@ -0,0 +1,38 @@
+import { addBreadcrumb, SeverityLevel } from "@sentry/react";
+import { createCoreLogger, Level } from '@openstax/ts-utils/services/logger';
+import { JsonCompatibleStruct } from "@openstax/ts-utils/routing";
+
+/**
+ * Creates a logger that creates breadcrumbs using Sentry.
+ * 
+ * More info: https://develop.sentry.dev/sdk/data-model/event-payloads/breadcrumbs/
+ */
+
+const serializeLevel = (level: Level): string =>
+  level === Level.Warn ? 'warning' : level;
+
+const serializeBreadcrumb = (level: Level, breadcrumb: JsonCompatibleStruct): {
+  type?: string;
+  level?: SeverityLevel;
+  event_id?: string;
+  category?: string;
+  message?: string;
+  timestamp?: number;
+  data?: { [key: string]: any };
+} => {
+  const { type, event_id, category, message, timestamp, data } = breadcrumb;
```

Comment:
you could take a rest parameter here and append the remaining stuff in the event to the `data` key. you'll have to do something to flatten it though because i think it only supports key value pairs. maybe something like this https://stackoverflow.com/a/74769002/14809536